JS Basics/Strings/Problem_2/index.js

```diff
@@ -0,0 +1,19 @@
+function correctBrackets(str) {
+    let bracketsArr = [];
+
+    for(let i = 0; i < str.length; i++) {
+        if(str[i] == '(') {
```

Comment:
Extract all hardcoded text in a const variable inside the function deffinition.
Open/Close brackets
Correct/Incorrect strings